packages/react-native-bridge/android/publish-aztec-and-bridge.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# 1. Verify the version argument is passed. We use the same version for react-native-aztec and react-native-bridge libraries.
+VERSION=$1
+if [[ -z $VERSION ]]; then
+    echo "This script requires the Bintray version to be passed as an argument."
+    echo "Example usage: './publish-aztec-and-bridge.sh \$VERSION'"
+    exit 1
+fi
+
+# 2. Check if we can call gradlew successfully
+./gradlew --help > /dev/null 2>&1
+gradlew_exists=$?
+if [ $gradlew_exists -ne 0 ]; then
+    echo "Gradle wrapper is not available. Make sure to run this script from '/packages/react-native-bridge/android'"
+    exit 1
+fi
+
+# 3. Clean and copy the bundle
+./gradlew clean -q
+mkdir -p react-native-bridge/build/assets
+cp ../../../../bundle/android/App.js ./react-native-bridge/build/assets/index.android.bundle
+
+if [ $? -ne 0 ]; then
+    echo "Make sure to run 'npm install' & 'npm run bundle:android' from the root folder of the project before running this script."
```

Comment:
Would it be better to include those commands in this script so that way we know that everything is completely up-to-date?

Reply:
Thinking about this some more, this script will actually only work if it is run from a Gutenberg repo that is "inside" gutenberg-mobile. Perhaps we should include a check for that. I'm not sure about the best approach here. 🤔

Reply:
> Would it be better to include those commands in this script so that way we know that everything is completely up-to-date?

I don't feel too comfortable with that idea. The main reason is I don't like when a script does more than what it says it'll do. In this case, I think this script should only publish aztec and bridge libraries. If we really need something like that, I'd rather add a second script in the `gutenberg-mobile` repo that calls this script after it sets everything up. However, I am not sure if that'll actually be useful since this script shouldn't really be run by developers. Let me know if you feel otherwise.

Aside from that main reason, this would require us to run commands from a parent directory, which again is not something I particularly like to do.

> Thinking about this some more, this script will actually only work if it is run from a Gutenberg repo that is "inside" gutenberg-mobile. Perhaps we should include a check for that. I'm not sure about the best approach here. 🤔

I think [this comment](https://github.com/WordPress/gutenberg/pull/29488#discussion_r589054816) also covers this issue, so I'll respond to it there.

Reply:
>  I am not sure if that'll actually be useful since this script shouldn't really be run by developers

Ah, I didn't realize this wasn't for use by developers. If a developer wanted to create an apk that included their changes in the js bundle, how would they go about doing that (i.e., if they wanted to send an apk to a designer for early feedback)?